<launch>
    

    <arg name="arg_json_file_path" default="/home/ding/backup/dataset/haibo/data/0313_2/global_map.json"/>
    <arg name="arg_pcd_file_path" default="/home/ding/backup/dataset/haibo/data/0313_2/transformed_map.pcd"/>
    <arg name="arg_point1_lat"  default="31.1377397" />
    <arg name="arg_point1_lon"  default="118.1788036" />


    <param name="point1_x" type="double" value="-12.7125101" />
    <param name="point1_y" type="double" value="105.8936768" />
    <param name="transform_from_llh" type="bool" value="true" />
    <param name="point1_lat" type="double" value="$(arg arg_point1_lat)" />
    <param name="point1_lon" type="double" value="$(arg arg_point1_lon)" />
    <!-- <param name="json_file_path" type="string" value="/home/ding/backup/code/haibo_lt_mapping/src/json_test/src/BB95_global_map.json" /> -->
    <param name="json_file_path" type="string" value="$(arg arg_json_file_path)" />
    <param name="json_file_path2" type="string" value="/home/ding/backup/dataset/haibo/data/0313_2/global_map.json" />
    <param name="pcd_file_path" type="string" value="$(arg arg_pcd_file_path)" />

    <node pkg="json_test" type="json_test" name="json_test" output="screen"/>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find json_test)/rviz/test.rviz" />
    </group>

</launch>
